src/main/java/com/google/devtools/build/lib/remote/RemoteModule.java

```diff
@@ -683,7 +683,7 @@ public OutputService getOutputService() {
 
   @Override
   public Iterable<Class<? extends OptionsBase>> getCommandOptions(Command command) {
-    return ImmutableList.of("build", "test", "fetch").contains(command.name())
+    return ImmutableList.of("build", "fetch", "query", "sync", "test").contains(command.name())
```

Comment:
This is not correct. It wasn't correct before this change either. The "test" command automatically inherits all "build" options, so doesn't need to be listed here.

Reply:
Do you want to send a PR that fixes it?